dependencies.props

```diff
@@ -17,15 +17,18 @@
   </PropertyGroup>
 
   <PropertyGroup>
+    <MicrosoftNETCorePlatformsPackageVersion>2.1.0-preview1-25824-02</MicrosoftNETCorePlatformsPackageVersion>
     <MicrosoftPrivateCoreFxNETCoreAppPackageVersion>4.5.0-preview1-25824-02</MicrosoftPrivateCoreFxNETCoreAppPackageVersion>
     <MicrosoftPrivateCoreFxUAPPackageVersion>4.6.0-preview1-25824-02</MicrosoftPrivateCoreFxUAPPackageVersion>
-    <PlatformPackageVersion>2.1.0-preview1-25824-02</PlatformPackageVersion>
     <MicrosoftNETCoreRuntimeCoreCLRPackageVersion>2.1.0-preview1-25824-51</MicrosoftNETCoreRuntimeCoreCLRPackageVersion>
-    <MicrosoftNETCoreRuntimeJitPackageVersion>$(MicrosoftNETCoreRuntimeCoreCLRPackageVersion)</MicrosoftNETCoreRuntimeJitPackageVersion>
-    <MicrosoftNetNativeCompilerPackageVersion>2.0.0-preview-25524-00</MicrosoftNetNativeCompilerPackageVersion>
-    <NETStandardVersion>2.1.0-preview1-25825-02</NETStandardVersion>
-    <DiaSymReaderNativeVersion>1.4.1</DiaSymReaderNativeVersion>
-    <WcfVersion>4.5.0-preview2-25825-02</WcfVersion>
+    <NETStandardVersion>2.1.0-preview1-25824-08</NETStandardVersion>
```

Comment:
This needs to be `NETStandardLibraryPackageVersion`

Reply:
Also - you are downgrading the version.

Reply:
I had to keep rebasing to deal with conflicts I must have messed this merge up the last time I did it. I will fix it.